### DISCUSSION TITLE
please help with search

### DISCUSSION THREAD

**Discussion by prankousky** [firefly-iii/firefly-iii#5424] [Firefly III]
Hi everybody,

I imported quite some data to the wrong account recently. They have been imported to the correct account (let's call it `801` / bank a, but *also* **falsly** to `402` / bank b).

They do actually belong to 801, so I am trying to find them via search, thus being able to delete them.

I tried `destination_account_nr_is:801`, which works. It will display **all** transactions in account a. Then I tried `destination_account_is:bankb`, which will show **all** transactions in account b. What I need is all transactions **in** bank b with number `801` (= bank a).

`destination_account_nr_is:801 destination_account_is:bankb` will not display anything. I also tried `destination_account_nr_is:801 AND destination_account_is:bankb`, but that won't show anything, either. 

Can I still find these false duplicates and only delete them from `bank b`- **keeping them** in bank a? I am using `firefly-iii-fints` to import data from my online banking accounts. Usually I make sure to pay extra attention to detail, but this time I must have imported bank b to the firefly-iii account for bank a. Which resulted in a complete mess.

Thank you in advance for your help :)

**Comment by JC5** [comment]
Please think of a descriptive title this is annoying.
